.github/workflows/ci.yml

```diff
@@ -11,7 +11,6 @@ on:
       - "*"
     branches:
       - main
-      - release/*
```

Comment:
The CI [failure](https://github.com/ansys/pyfluent/actions/runs/7570174881/job/20639780400?pr=2369#step:2:132) was due to a failing branch name style action. On pushing release/* branch, the branch name is evaluated as `refs/heads/release/0.19.0` instead of `release/0.19.0`.

Reply:
Thank you @mkundu1